boards/samr21-xpro/Makefile.features

```diff
@@ -9,6 +9,9 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
 
+# Put other features for this board (in alphabetical order)
+FEATURES_PROVIDED += riotboot
```

Comment:
As I had it tested do you want to add it also to saml21-xpro board?

Reply:
Yes, I can do that myself.

Reply:
Done.